src/index.js

```diff
@@ -1,49 +1,192 @@
 const {
   BaseKonnector,
   requestFactory,
-  saveFiles,
-  addData
+  saveBills,
+  updateOrCreate,
+  scrape,
+  signin
 } = require('cozy-konnector-libs')
-const request = requestFactory({ cheerio: true })
+const log = require('cozy-logger')
+const pdf = require('pdfjs')
+const html2pdf = require('./html2pdf')
+const moment = require('moment')
 
-const baseUrl = 'http://books.toscrape.com'
+const baseUrl = 'https://www.netflix.com'
+const NBSPACE = '\xa0'
+const DEBUG = false
```

Comment:
Not used on every `request()` call... on purpose?

Reply:
Not really